stdlib/2and3/winsound.pyi

```diff
@@ -0,0 +1,24 @@
+
+import sys
+from typing import Optional, Union
+
+if sys.platform == "win32":
+    SND_FILENAME: int
+    SND_ALIAS: int
+    SND_LOOP: int
+    SND_MEMORY: int
+    SND_PURGE: int
+    SND_ASYNC: int
+    SND_NODEFAULT: int
+    SND_NOSTOP: int
+    SND_NOWAIT: int
+
+    MB_ICONASTERISK: int
+    MB_ICONEXCLAMATION: int
+    MB_ICONHAND: int
+    MB_ICONQUESTION: int
+    MB_OK: int
+
+    def Beep(frequency: int, duration: int) -> None: ...
+    def PlaySound(sound: Optional[Union[str, bytes, bytearray, memoryview]], flags: int) -> None: ...
```

Comment:
PlaySound can accept a filename (I assume `str` is used for this :)) -- maybe add a `Path` to the `Union`?

Reply:
Actually `sound` must be a `str` or `None`:

```python
>>> winsound.PlaySound(b"", 0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'sound' must be str or None, not 'bytes'
```

(Tried with Python 3.8.0.)

Sidenote: `bytes` is equivalent to `Union[bytes, bytearray, memoryview]` in arguments.

Reply:
My apologies, thank you for the correction @srittau